happy-travel/edo
# Feature/aa 1875 remove account creation for read only

Issue: https://github.com/happy-travel/agent-app-project/issues/1875

Test cases: 
 1) Create root agency and verify it to ReadOnly stage
 2) Check if virtual accounts weren't created
 3) Check if agency in ReadOnly stage works properly without accounts
 4) Check search stages
